scout/server/blueprints/institutes/controllers.py

```diff
@@ -422,28 +424,65 @@ def _sort_cases(data, request, all_cases):
     return all_cases
 
 
-def cases(store, request, institute_id):
-    """Preprocess case objects.
+def export_case_samples(institute_id, filtered_cases) -> Response:
+    """Export to CSV file a list of samples from selected cases."""
+    EXPORT_HEADER = [
+        "Sample ID",
+        "Sample Name",
+        "Analysis",
+        "Phenotype",
+        "Sex",
+        "Tissue",
+        "Case Name",
+        "Case ID",
+        "Analysis date",
+        "Case Status",
+        "Track",
```

Comment:
What else?

Reply:
Default panel is rather useful for a quick glance. Research open seems straightforward. Parenthood confirmed could be good. Rank model versions. Genome build? Sex confirmed? Pred ancestry? Maybe HPO terms?

The list could potentially be long: slightly hard to reach features e.g. from the `delivery_report` or `exe_ver`/`reference_info`, like what pipeline, pipeline version would likely be appreciated. From delivery report or chanjo/2 average coverage...

But all in all, a good start is a good start, and we can always expand!